### ISSUE TITLE
Telegram sends an alert once a second instead of once for each alarm triggered.

### ISSUE THREAD

**Issue by HeinB** [thingsboard/thingsboard#717]
I am being spammed for as long as the alarm is active, rather than just getting one message per event.  My setup:
For the Processor bit: Alarm trigger expression: temperature >= 24 and Alarm clear expression: temperature < 23
Alarm Type: alarm -> I set this to the value "alarm", is this ok?
Propagate alarm is checked and I have no data in the Alarm Details json field (I tried with the temoerature in there, same issue)
New Alarm on eavh event is NOT checked.

I chose plugin as the Telegram Plugin
Body template has:
{"chat_id": "12345 (but my actual id):, "text": "Temperature is high! $temperature.valueAsString degrees!"}

It works and sends the correct message, but keeps on sending a message every second until the temperature drops to the Alarm clear expression value.  Please help!

**Comment by nmenoni** [comment]
You have to set "isNewAlarm" flag. I don´t remember exactly where, I thinks
it is in the plugin

2018-04-13 10:52 GMT-03:00 HeinB <notifications@github.com>:

**Comment by HeinB** [comment]
Thanks for the response.  The isNewAlarm is an option in the plugin action for the Email plug-in (and works fine). The rest API plug-in does not have this option to set in the plug-in action.  Not sure how to set this?

**Comment by AndriiHavrylyak** [comment]
Hi, I have some problems with the alarm widget. If the temperature> 20 this alarm. I want to see in the journal avars when this alarm occurred. But the alarm is formed only once, and everything is not added more. The new alarm will be added only if I delete the old one.HELP anyone know ,how to see all alarm list?

**Comment by vparomskiy** [comment]
An alarm can be updated. Please use this tutorial as a reference:
https://thingsboard.io/docs/user-guide/rule-engine-2-0/tutorials/create-clear-alarms-with-details/#step-2-createupdate-alarm

**Comment by vparomskiy** [comment]
Duplicate of [#1140](https://github.com/thingsboard/thingsboard/issues/1140)
Close
